### ISSUE TITLE
Add separate AirPrice call so that LFS results can be confirmed and additional info retrieved

### ISSUE THREAD

**Issue by wesnolte** [Travelport-Ukraine/uapi-json#394]
Results from LFS are good but they are cached so a call to AirPrice is sometimes required to confirm availability and price before booking. LFS results also only have limited info about certain things e.g. baggage, so info from AirPrice is required if we want to display accurate details such as those before booking.

Please could you create a separate AirPrice function that could then be used prior to Booking.

**Comment by engrashid** [comment]
I agree with  @wesnolte . This will also help to add information  to price response as needed . As for some booking plating carrier needed to add in price info section .

**Comment by creduo** [comment]
Did #405 helped this issue?
